saleor/core/middleware.py

```diff
@@ -1,6 +1,7 @@
 import logging
 
 from django.conf import settings
+from django.contrib.sites import models as sites
```

Comment:
Why `import models as sites`? Can't we just explicitly import what we need in this module? From what I see we use only SiteManager here, we can import only that.

Reply:
This is not done yet but basicly yes, this will be changed